docs/products/kafka/kafka-connect/howto/s3-sink-connector-aiven.rst

```diff
@@ -0,0 +1,115 @@
+Create a S3 sink connector by Aiven
+===================================
+
+The Apache Kafka Connect® S3 sink connector by Aiven enables you to move data from Aiven for Apache Kafka cluster to Amazon S3 for long term storage.
+
+.. Note::
+
+    There are two version of S3 sink connector available with Aiven for Apache Kafka Connect®: One is developed by Aiven, another developed by Confluent. This article use Aiven version while the S3 sink connector by Confluent in a `dedicated page <https://help.aiven.io/en/articles/2413736-aiven-kafka-s3-sink-connector-by-confluent>`_.
+
+Prerequisites
+-------------
+
+To setup the S3 sink connector by Aiven, you need an Aiven for Apache Kafka service :doc:`with Apache Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+
+Furthermore you need to follow the :doc:`required setup steps <s3-sink-prereq>` and collect the following information about the target S3 bucket upfront:
```

Comment:
I believe ....to follow the steps :doc:`to prepare AWS account <s3-sink-prereq>`.... is a bit better describes the content of that small article.